src/main/java/org/jabref/gui/git/ExternalChangesResolverViewModel.java

```diff
@@ -0,0 +1,55 @@
+package org.jabref.gui.git;
+
+import java.util.List;
+
+import javax.swing.undo.UndoManager;
+
+import javafx.beans.binding.BooleanExpression;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.ObservableList;
+
+public class ExternalChangesResolverViewModel {
+    private StringProperty selectedChange;
+
+    public ExternalChangesResolverViewModel(List<GitChange> changes, UndoManager undoManager) {
+        this.selectedChange = new SimpleStringProperty();
+    }
+
+    public boolean areAllChangesResolved() {
+        return false;
+    }
+
+    public ObservableList<GitChange> getVisibleChanges() {
+        return null;
```

Comment:
NEVER return null. At least try to.

Return an empty list.

---

When is the class used? I don't see an update of some variable "visibleChanges" here...

Is this class a left over?